src/main/template/main.handlebars

```diff
@@ -82,40 +101,52 @@
+                                          <strong>Client ID:</strong> {{clientId}}<br/>
+                                          {{#if baseUrl}}<strong>Base URL:</strong> {{baseUrl}}{{/if}}
                                       </p>
+                                      {{#if accessToken}}
+                                        <h5 id="your-current-bearer-token">Your Current Bearer Token</h5>
+                                        <p>When you logged in, a Bearer Token (JWT) was automatically generated for you. This token will automatically be added to requests made suing this page. You can also use this token for your own testing purposes for as long as it’s valid. But be careful! Treat this token like a password.</p>
+                                        <p><code style="word-wrap: break-word">{{accessToken}}</code></p>
+                                      {{/if}}
                                     </div>
                                 </div>
                                 <div class="samples">
                                     <span class="model-signature">
                                         <div>
                                             <div role="tabpanel" class="guide">
-                                                <ul class="nav nav-pills" role="tablist">
-                                                    <li role="presentation" class="active">
-                                                        <a href="#example_auth_curl"role="tab" data-toggle="tab">Example Curl</a>
-                                                    </li>
-                                                </ul>
+                                                <h5>Example cURL request</h5>
                                                 <div class="tab-content">
                                                     <div role="tabpanel" class="tab-pane active" id="example_auth_curl">
-                                                        <div class="snippet">
-<pre><code class="bash"><span class="dollar"></span>curl "https://cimpress.auth0.com/oauth/ro" \
-    -d client_id=<span class="string">"{{clientId}}"</span> <span class="comment"># for {{info.title}}</span> \
-    -d username=<span class="string">"Your Email"</span> \
-    -d password=<span class="string">"Your Password"</span> \
-    -d connection=<span class="string">"default"</span> \
-    -d scope=<span class="string">"openid email app_metadata"</span>
+                                                        <!-- Authentication code sample (NG-5085) -->
+                                                        <div class="snippet"><pre><code class="bash"><span class="comment"># Get an auth token for the API with client_id</span>
+<span class="comment"># representing “{{info.title}}”</span>
+<span class="dollar"></span><span class="built_in">curl</span> "<span style="text-decoration: underline;">https://cimpress.auth0.com/oauth/ro</span>" \
```

Comment:
<img width="1186" alt="screen shot 2017-02-01 at 7 55 47 pm" src="https://cloud.githubusercontent.com/assets/693878/22532901/7d9ca908-e8b8-11e6-9acf-647440f9b1f6.png">

Here's a screenshot to illustrate this part of the diff. This is the part of the PR that I'm requesting a review of for the ticket NG-5085.